modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1443,8 +1438,6 @@ public void processAttribute(String prefix, String localName, String value)
 
         @Override
         public void processEndElement() throws IOException {
-            super.processEndElement();
```

Comment:
this suggests a specific pattern, even though there is probably no development expected in this class.
i'd keep it as is, and revert making the method abstract in the base class.

Reply:
I agree. This sort of change goes beyond what I expect to see in a cleanup pass.

Reply:
I've reverted these changes and documented the exception instead.

Also resolved all other conversations with the same root cause.